src/chains/ethereum/ethereum/tests/api/eth/getTransactionReceipt.test.ts

```diff
@@ -0,0 +1,74 @@
+import getProvider from "../../helpers/getProvider";
+import assert from "assert";
+import EthereumProvider from "../../../src/provider";
+
+describe("api", () => {
+  describe("eth", () => {
+    describe("getTransactionReceipt", () => {
+      let provider: EthereumProvider;
+      let logger;
+
+      beforeEach(async () => {
+        // create a logger to test output
+        logger = {
+          clearLoggedStuff: function () {
+            this.loggedStuff = "";
+          },
+          loggedStuff: "",
+          log: function (message) {
+            if (message) {
+              this.loggedStuff = this.loggedStuff.concat(message);
+            }
+          }
+        };
+        provider = await getProvider({ logging: { logger } });
+      });
+
```

Comment:
I made suggestions to check the logs in each test below, so this change is probably useful:

```suggestion

      afterEach(() => {
        logger.clearLoggedStuff();
      });
      
```